dissect/target/plugin.py

```diff
@@ -1258,6 +1258,18 @@ def os(self) -> str:
         """
         raise NotImplementedError
 
+    @internal
+    def misc_user_paths(self) -> Iterator[tuple[str, tuple[str, str] | None]]:
+        """Yields miscellaneous user paths and user keys.
+
+        Example:
+            ("%windir%/ServiceProfiles/LocalService", ("sid", "S-1-5-19"))
```

Comment:
This example doesn't reflect reality, where we yield resolved paths instead of paths containing environment variables.